src/System.Net.Mail/tests/Functional/SmtpExceptionTest.cs

```diff
@@ -86,24 +86,6 @@ public void TestConstructorWithStringArgument()
         }
 
         [Fact]
-        public void TestConstructorWithSerialization()
-        {
-            string msg = "MESSAGE";
-            Exception inner = new ArgumentException("whatever");
```

Comment:
ArgumentException isn't marked Serializable, but that test passes. Perhaps DCS is using POCO serialization with it?